protocol/errors.json

```diff
@@ -61,6 +61,26 @@
 	"50002": "internal connection error",
 	"50003": "timeout error",
 
+	/* frontend connection-related */
+	"55000": "internal error (uncategorised)",
+
+	/* frontend connection-related */
+	"55100": "internal error (generic connection error)",
+
+	/* frontend channel-related */
+	"55200": "internal error (generic channel or attachment error)",
+	"55201": "internal error (channel rpc error)",
+
+	/* core channel-related */
+	"55300": "internal error (generic message processing error)",
+	"55301": "internal error (core rpc error)",
+
+	/* core webhook/reactor-related */
+	"55400": "internal error (generic reactor error)",
+
+	/* persistence-related */
```

Comment:
> Would this include History persistence, or persistence more generally?

It would be generally, and I thinking of errors in the cassandra/redis layers where you don't know whether you're doing something for history or something else.